package.json

```diff
@@ -127,7 +127,7 @@
     "husky": "^8.0.3",
     "mocha": "^10.8.2",
     "mongodb": "^6.3.0",
-    "mssql": "^10.0.1",
+    "mssql": "^11.0.0",
```

Comment:
There should be corresponding changes to the package-lock.json after running npm install with this change. Can you commit them here?

Reply:
updated.